### ISSUE TITLE
Reduntant text after theme name

### ISSUE THREAD

**Issue by AghaSaad04** [phpmyadmin/phpmyadmin#15068]
### Describe the bug

I have switched the themes of phpmyadmin and noticed something different in theme: metro. There's something "scheme: win" (basically it is a scheme color and I don't know why is it there and not in other themes and it is not useful information) written after theme name which is not in the other two themes. I think this should be removed.

### To Reproduce

1. Change theme to Metro
2. You will see the "Scheme:win" thing

### Expected behavior
I was expecting the consistency in all the themes.

### Screenshots

![image](https://user-images.githubusercontent.com/36513474/54870579-389c2300-4dca-11e9-8ca9-0ee78bddbd75.png)

### Server configuration

- Operating system: Windows
- Web server: XAMPP
- PHP version: 7.3.0
- phpMyAdmin version:  5.0.0-dev

### Client configuration

- Browser: Firefox, Chrome
- Operating system: Windows

**Comment by mauriciofauth** [comment]
Metro theme have different color schemes. See:

https://github.com/phpmyadmin/phpmyadmin/blob/b247fbedfe9ccfbcf20ca75451bd34eecea26a87/themes/metro/scss/_variables.scss#L10-L14

However there is no way to change the color scheme through the GUI.

**Comment by AghaSaad04** [comment]
Yeah, you are right Sir @mauriciofauth, but as we cannot change that color scheme from GUI, then I think there's no need for that.

**Comment by apoorvkhare07** [comment]
@mauriciofauth Should we add the feature of changing the color scheme through GUI ?

**Comment by mauriciofauth** [comment]
I don't know if adding a option to change the color scheme through GUI would be a good idea, because only a few themes have color schemes.
We should at least need to make changing color schemes more easy, like replacing CSS files.

**Comment by joho1968** [comment]
> Metro theme have different color schemes. See:
> [..] 
> However there is no way to change the color scheme through the GUI.

I can't seem to find any reference in the 5.x documentation on how to set the color scheme either, but I may be looking in the wrong place. Or is something like "metro-mono" the implied syntax? If so, I think this should be stated in the documentation.

**Comment by williamdes** [comment]
To learn about changing schemes checkout: https://github.com/phpmyadmin/phpmyadmin/issues/16969#issuecomment-864187054
